pkg/mgrconfig/config.go

```diff
@@ -26,6 +26,8 @@ type Config struct {
 	KernelSrc string `json:"kernel_src,omitempty"`
 	// Location of the driectory where the kernel was built (if not set defaults to KernelSrc)
 	KernelBuildSrc string `json:"kernel_build_src"`
+	// Map of modules and their symbol files.
+	KernelModules json.RawMessage `json:"kernel_modules"`
```

Comment:
This can be just a map, no need to unmarshal separately. Maps are natively supported by json package.
What is "symbol files"? How users are supposed to generate this? Can this be auto-inferred?